backend/installGlobalHook.js

```diff
@@ -57,6 +65,33 @@ function installGlobalHook(window: Object) {
           this._listeners[evt].map(fn => fn(data));
         }
       },
+      // Fiber-only:
+      supportsFiber: true,
```

Comment:
No:

```js
if (
  typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ !== 'undefined' &&
  typeof __REACT_DEVTOOLS_GLOBAL_HOOK__.supportsFiber
) {
```

Necessary so that Fiber doesn't try to attach to non-Fiber DevTools and crash them.